xmppserver/src/main/java/org/jivesoftware/util/SystemProperty.java

```diff
@@ -119,7 +119,7 @@
         if (StringUtils.isEmpty(value)) {
             return Stream.empty();
         }
-        final List<String> strings = Arrays.asList(value.split(","));
+        final List<String> strings = Arrays.asList(value.trim().split("[\\s,]+"));
```

Comment:
I am curious what happens with a string like so "1 , 2 ,  3" ?  Could that be added as a test?

Reply:
Force-pushed an update that includes this in a new unit test called `willCreateAListOfCommaWhitespaceSeparatedString2`. The test passes.